demos/2022.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: demos
+title: Community Demos
```

Comment:
Should include the year to differentiate pages? If so, we should do the same with the 2021 page.
```suggestion
title: Community Demos 2022
```